pkg/services/dashboards/dashboard_service.go

```diff
@@ -15,13 +15,15 @@ import (
 type DashboardService interface {
 	SaveDashboard(dto *SaveDashboardDTO) (*models.Dashboard, error)
 	ImportDashboard(dto *SaveDashboardDTO) (*models.Dashboard, error)
+	DeleteDashboard(dashboardId int64, orgId int64) error
```

Comment:
I also added this function to service so it can be called instead of direct query with the `bus`

Reply:
Oh missed these. Have commented about these in my latest review. Resolving these...